mcp/include/ezstep.h

```diff
@@ -140,82 +150,88 @@ struct ezbus {
                                 // with ezbus
 };
 
-/* initialize a struct ezbus. Needed for all other EZbus funuctions
+
+/** @brief initialize a struct ezbus. Needed for all other EZbus funuctions
```

Comment:
I'm not exactly a Doxygen expert, but I suspect that the lack of an empty line separating the summary from the rest will make Doxygen interpret thede lines as the part of the summary.

Reply:
They're params, I've added @param